async-development/js/modules/saveToLS.js

```diff
@@ -0,0 +1,30 @@
+const WEATHER_KEY = 'weather';
+
+export const saveToLocalstorage = (data) => {
+    const dataFromLocalstorage = localStorage.getItem(WEATHER_KEY);
+    if (dataFromLocalstorage) {
+        const weather = JSON.parse(dataFromLocalstorage);
+        const { location, current } = data;
+        weather[location.name.toLowerCase()] = getWeatherObj(location, current);
+        localStorage.setItem(WEATHER_KEY, JSON.stringify(weather));
+    } else {
+        const { location, current } = data;
+        const weather = {[location.name.toLowerCase()]: getWeatherObj(location, current)};
+        localStorage.setItem(WEATHER_KEY, JSON.stringify(weather));
+    }
+}
+
+const getWeatherObj = (location, current) => {
+    return {
+        "name": location.name,
```

Comment:
nitpick: no need to wrap with quotes
```suggestion
        name: location.name,
```